src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/RuntimeMetrics.cs

```diff
@@ -146,12 +146,11 @@ static RuntimeMetrics()
             unit: "{cpu}",
             description: "The number of processors available to the process.");
 
-        // TODO - Uncomment once an implementation for https://github.com/dotnet/runtime/issues/104844 is available.
-        //private static readonly ObservableCounter<double> s_processCpuTime = s_meter.CreateObservableCounter(
-        //    "dotnet.process.cpu.time",
-        //    GetCpuTime,
-        //    unit: "s",
-        //    description: "CPU time used by the process as reported by the CLR.");
+        private static readonly ObservableCounter<double> s_processCpuTime = s_meter.CreateObservableCounter(
+           "dotnet.process.cpu.time",
+           GetCpuTime,
+           unit: "s",
+           description: "CPU time used by the process as reported by the CLR.");
```

Comment:
Is the "as reported by the CLR" part necessary? Do we use that phrasing for other instruments? Does this not work on mono / does the runtime matter?

Reply:
+1

The time is reported by the OS. .NET runtime is not involved in tracking it in any way. The managed API is just a thin wrapper over the underlying OS API.

Reply:
This needs to be addressed outside this PR. Please look at https://github.com/open-telemetry/semantic-conventions/pull/1035 for detailed discussions around the semantic convention of all metrics we are exposing. Feel free to shime there and comment on any of the proposed one.

Reply:
Why is this metric special? eg there's one that says "The number of garbage collections that have occurred since the process has started" not "The number of garbage collections that have occurred since the process has started as reported by the CLR"... why did this one need that phrase if that one doesn't? As Jan said, it's also misleading.

Reply:
The description proposed in https://github.com/open-telemetry/semantic-conventions/pull/1035 for this metric is not good. I do not see the point in propagating the bad description in multiple places. We should include a good description here, and fix the open-telemetr PR to match

Reply:
are you proposing changing `CLR` with `operating system`?

Reply:
I would make it just `CPU time used by the process` and drop the `as reported` part completely.

There are other metrics that are tracked and reported by the operating system, like `WorkingSet`, and their description does not mention this detail either.

Reply:
Sounds good to me.